### DISCUSSION TITLE
Welcome to gopher-lua Discussions!

### DISCUSSION THREAD

**Discussion by yuin** [yuin/gopher-lua#369] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by serprex** [comment]
Hi, great project, I'm hoping to use gopher-lua as a scripting engine for peerdb (postgres ETL product)

I have some experience optimizing interpreter runtimes, outside of making optimized Befunge JITs [I got CPython 3.6 to use wordcode](https://bugs.python.org/issue26647), if you have any issues you'd like me to look into let me know
